### ISSUE TITLE
filters (fourier masking and real space masking)

### ISSUE THREAD

**Issue by geoffwoollard** [mjo22/cryojax#237]
Suggested filters: 

- [ ] Butterworth: https://en.wikipedia.org/wiki/Butterworth_filter
- [ ] Cosine: https://en.wikipedia.org/wiki/Raised-cosine_filter

**Comment by geoffwoollard** [comment]
These filters have some free parameters that affects when they decay and show sharply.

**Comment by mjo22** [comment]
Filters and masks exist in cryojax.image.operators. Adding a few more cosine masks based on the implementation in cisTEM in #238.

**Comment by mjo22** [comment]
@geoffwoollard given this is already in the code (although still not comprehensively), could you make this issue more specific?

**Comment by mjo22** [comment]
Closing, since this already exists. Would be good to open a new issue if there are specific additional filters / masks that are desired
